contracts/utils/FHESafeMath.sol

```diff
@@ -31,6 +29,9 @@ library FHESafeMath {
      * and `updated` will be the original value.
      */
     function tryDecrease(euint64 oldValue, euint64 delta) internal returns (ebool success, euint64 updated) {
+        if (!FHE.isInitialized(oldValue)) {
+            return (FHE.asEbool(false), oldValue);
+        }
```

Comment:
This is not correct since if `oldValue` and `delta` are both uninitialized, we would want success to be true.